src/base/memory_monitor_sr.cpp

```diff
@@ -20,18 +20,62 @@
  * memory_monitor_sr.cpp - Implementation of memory monitor module
  */
 
+#include <malloc.h>
+#include <cassert>
 #include <cstring>
 #include <algorithm>
-#include <cassert>
 
 #include "memory_monitor_sr.hpp"
 
+#ifndef HAVE_USR_INCLUDE_MALLOC_H
+#define HAVE_USR_INCLUDE_MALLOC_H
+#endif
```

Comment:
`memory_monitor_sr.cpp` has to include `malloc.h` because of `malloc_usable_size()`. But when user includes `malloc.h`, user has to define `HAVE_USR_INCLUDE_MALLOC_H`.

Reply:
I heard from you offline this is needed to compile later. So, I request you add this when you need to this to complie and discuss it. Because I think this macro only releated to re-define `struct mallinfo` when I read the manual.

Reply:
I removed the define. Let's talk about it later issue :)